deploy/operator.yaml

```diff
@@ -27,7 +27,7 @@ spec:
       labels:
         name: rocketmq-operator
     spec:
-      serviceAccountName: rocketmq-operator
+      serviceAccountName: {{ template "rocketmq-operator.serviceAccountName" . }}
```

Comment:
Files in `deploy` directory are for deploying by `kubectl` instead of helm, so templating is not supported. 
@drivebyer Pls also take a look, it seems some substitution operations should be added in `make manifest`.

Reply:
Sure!

Reply:
For now, After we execute `make manifest `, the templating in `chart` directory disappeared. 

I would find some way to do substitution in `make manifest.`

Reply:
Maybe we can generate chart by helmify, https://github.com/apache/rocketmq-operator/pull/212

Reply:
Finally got some time for this one. So basically, if I understand it right, and correct me if I'm not, that those `yaml` files in the `deploy` directory don't just serve one purpose which is to be copied to the `templates` directory, they are also being used directly by the `kubectl` command.

And to fix what I want to fix, something needs to be done in the `make manifest` process to properly *generate / create / substitute after copy / or any other ways possible* `yaml` files with helm templating contents in the `templates` directory while at the same time keep `yaml` files in the `deploy` directory free from helm templating contents.

The thing is, am I allowed to choose whatever means possible to achieve this or should I wait for some discussions to reach a conclusion and then go on with this PR?

Reply:
@sinloss IMO we should fix bugs in `make manifest` to generate correct files in deploy (for `kubectl`) and charts (for `helm`). Pls check if #212 can achieve it.